src/pages/add-lock-exception/AddLockExceptionForm/AddLockExceptionForm.js

```diff
@@ -0,0 +1,79 @@
+import i18n from '@dhis2/d2-i18n'
+import { SingleSelectField, SingleSelectOption } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './AddLockExceptionForm.module.css'
+import OrganisationUnitSelection from './OrganisationUnitSelection/OrganisationUnitSelection'
+import PeriodPicker from './PeriodPicker'
+
+const AddLockExceptionForm = ({
```

Comment:
This component is a refactor of https://github.com/dhis2/data-administration-app/blob/fd85975586ad812d67f88005a9db069d82b84d66/src/pages/lock-exception/AddLockExceptionForm.js